packages/flutter/test/material/expansion_tile_test.dart

```diff
@@ -163,7 +163,7 @@ void main() {
     await tester.pumpWidget(
       MaterialApp(
         theme: ThemeData(
-          accentColor: _accentColor,
+          colorScheme: ColorScheme.fromSwatch().copyWith(secondary: _secondaryColor),
```

Comment:
Neat trick with the fromSwatch(). I will have to remember that for my own tests in the future.